src/v2/config/xcm/XcmRepositoryConfiguration.ts

```diff
@@ -30,6 +31,7 @@ export const XcmRepositoryConfiguration: TypeMapping = {
   [Chain.PHALA]: PhalaXcmRepository,
   [Chain.BIFROST_POLKADOT]: BifrostXcmRepository,
   [Chain.BIFROST_KUSAMA]: BifrostXcmRepository,
+  [Chain.EQUILIBRIUM]: EquilibriumXcmRepository,
 };
```

Comment:
@alex-nax 
Please add `[Chain.GENSHIRO]: EquilibriumXcmRepository` here if you want to support Genshiro at this PR (and submit the required links). Otherwise, please remove Genshiro from this PR.

<img width="567" alt="Screenshot 2023-03-07 at 12 48 27 PM" src="https://user-images.githubusercontent.com/92044428/223325180-9eb89565-2dd9-487e-b348-217e76620f8e.png">